bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/admin/TestColumnDescriptorAdapter.java

```diff
@@ -198,24 +199,24 @@ public void testGCruleMaxVersion(){
   public void testAdaptWithColumnFamilyForMaxAge(){
     int ttl = 86400;
     GCRule maxAgeGCRule = GCRULES.maxAge(Duration.ofSeconds(ttl));
-    ColumnFamily columnFamily = ColumnFamily.newBuilder().setGcRule(maxAgeGCRule.toProto()).build();
-    HColumnDescriptor actual = adapter.adapt(FAMILY_NAME, columnFamily);
+    HColumnDescriptor actual = adapter.adapt(columnFamily(maxAgeGCRule));
     Assert.assertEquals(ttl, actual.getTimeToLive());
   }
 
-  private static ColumnFamily columnFamily(GCRule rule) {
-    return ColumnFamily.newBuilder().setGcRule(rule.toProto()).build();
+  private static com.google.cloud.bigtable.admin.v2.models.ColumnFamily columnFamily(GCRule rule) {
```

Comment:
Please add a TODO comment to remove this method, and the methods that create `GCRule` to take in a `ColumnFamily` instead of converting a model -> proto -> model